client/a8c-for-agencies/sections/sites/needs-setup-sites/site-configurations-modal/use-random-site-name.tsx

```diff
@@ -49,3 +51,6 @@ export const useRandomSiteName = () => {
 
 	return { randomSiteName, isRandomSiteNameLoading };
 };
+
+// https://public-api.wordpress.com/rest/v1.1/domains/suggestions?http_envelope=1&query=test&quantity=1&include_wordpressdotcom=true&include_dotblogsubdomain=false&vendor=dot
+// https://public-api.wordpress.com/rest/v1.1domains/suggestions?http_envelope=1&query=test&quantity=1&include_wordpressdotcom=true&include_dotblogsubdomain=false&vendor=dot
```

Comment:
Probably some leftovers from development.

Reply:
Oh! Thank you for catching that!